duckduckgo_search/ddg.py

```diff
@@ -117,7 +117,12 @@ def get_ddg_page(page):
 
     # download documents
     if download:
-        keywords = keywords.replace('"', "'")
+        keywords = (
+            keywords.replace('"', "'")
+            .replace("site:", "")
```

Comment:
the description isn't that accurate, as keywords containing `site:` doesn't introduce bugs on file writes. I suspect `/` (posix) and `\\` (windows) are both scenarios which need slashes clean up from the filename / filepath before we attempt to `open(<filename>)`.

@deedy5  do people use this package on both posix and windows file systems?